core/src/main/kotlin/model/Documentable.kt

```diff
@@ -57,8 +62,13 @@ class Property(
     override val receiver: Parameter?,
     override val expected: PlatformInfo?,
     override val actual: List<PlatformInfo>,
-    override val extra: MutableSet<Extra> = mutableSetOf()
-) : CallableNode() {
+    override val extra: MutableSet<Extra> = mutableSetOf(),
+    val type: TypeWrapper?,
+    val accessors: List<Function>,
+    val isVar: Boolean = false,
+    override val sourceLocation: String? = null,
+    override val visibility: Visibility
```

Comment:
What if visibility is different between platforms?

Reply:
Changed to map.